src/include/uapi/user/tokens.h

```diff
@@ -111,6 +111,9 @@
 #define SOF_TKN_INTEL_DMIC_PDM_CLK_EDGE		705
 #define SOF_TKN_INTEL_DMIC_PDM_SKEW		706
 
+/* Tone */
+#define SOF_TKN_TONE_SAMPLE_RATE		800
```

Comment:
is this missing from kernel headers ? If so we need to add it as a separate PR ASAP @plbossart fyi.